mastodon/src/main/java/org/joinmastodon/android/ui/displayitems/WarningFilteredStatusDisplayItem.java

```diff
@@ -55,7 +51,11 @@ public Holder(Context context, ViewGroup parent){
 		@Override
 		public void onBind(WarningFilteredStatusDisplayItem item) {
 			filteredItems = item.filteredItems;
-			text.setText(item.parentFragment.getString(R.string.sk_filtered, item.applyingFilter.title));
+			if(item.applyingFilter instanceof AltTextFilter){
+				text.setText(item.parentFragment.getString(R.string.sk_filtered,item.parentFragment.getString(R.string.sk_no_alt_text)));
+			}else{
+				text.setText(item.parentFragment.getString(R.string.sk_filtered, item.applyingFilter.title));
+			}
```

Comment:
```suggestion
String title = item.applyingFilter instanceof AltTextFilter ? item.parentFragment.getString(R.string.sk_no_alt_text) : applyingFilter.title;
text.setText(item.parentFragment.getString(R.string.sk_filtered, text));
```
Both branches share a larger amount of code, so it is easier to read when using a conditional.